gravitational/gravity
# Update system pods check

## Description

This PR updates the system pods check. Some users have experienced a heavy load on etcd  if there are a large number of pods in the cluster. More details in https://github.com/gravitational/gravity/issues/2244. 

Updated system pods check will help reduce load on etcd if there are a large number of pods in user namespaces.

## Type of change


* Bug fix (non-breaking change which fixes an issue)

## Linked tickets and other PRs



* Refs https://github.com/gravitational/gravity/issues/2244

* Requires https://github.com/gravitational/satellite/pull/278, https://github.com/gravitational/planet/pull/772

## TODOs



## Testing done

**Test coredns CrashLoopBackOff**
- Update coredns configmap
- Restart coredns pod
- Verify system-pod warning for coredns pod
